glide.yaml

```diff
@@ -0,0 +1,268 @@
+package: kubevirt.io/kubevirt
+ignore:
+- github.com/go-kit/kit/examples
+- github.com/googleapis/gnostic/plugins
+- github.com/go-kit/kit/log/levels
+- github.com/go-kit/kit/metrics/influxdb
+- github.com/go-kit/kit/service
+- k8s.io/apimachinery/pkg/apis/meta/v1alpha1
+- code.google.com/p/go-charset
+import:
+- package: github.com/emicklei/go-restful
+  version: 858e58f98abd32bc4d164a08e8d7ac169ae876e2
+  subpackages:
+  - log
+  - swagger
+- package: golang.org/x/net
+  version: 054b33e6527139ad5b1ec2f6232c3b175bd9a30c
+  subpackages:
+  - internal/socket
+- package: github.com/go-kit/kit
+  version: fe6fe28ba0d54b39f27e79cddba4911b7e4fffc7
+  subpackages:
+  - log
+- package: google.golang.org/grpc
+  version: d3ddb4469d5a1b949fc7a7da7c1d6a0d1b6de994
+  subpackages:
+  - metadata
+  - grpclb/grpc_lb_v1
+  - keepalive
+  - stats
+  - status
+  - tap
+- package: k8s.io/apimachinery
```

Comment:
Oh wait. How are they pulled in? Does it read the godeps file in the repo? If it just pulls in the latest commit from master, that is probably not a good option, because we have to keep all the kubernetes dependencies across repos synchronized, and they are syncing every day ....